test/helpers/middleware/appEventInterceptor.js

```diff
@@ -0,0 +1,78 @@
+import _ from 'lodash'
```

Comment:
looks like this might have been copied in from the futarchy-app and checked in by accident?